plugins/Satellites/src/gui/SatellitesDialog.cpp

```diff
@@ -460,7 +463,7 @@ void SatellitesDialog::updateSatelliteData()
 	Q_ASSERT(SatellitesMgr);
 	Vec3f mColor, oColor, iColor;
 
-	// set default
+	// set default // XXX oy-pasta
```

Comment:
Huh?

Reply:
typo, should be "copy-pasta". the color setting code is indeed copied further down the file. 
refactored.